.github/workflows/ci.yml

```
@@ -47,19 +51,23 @@
+            echo "No changed files to lint."
+          elif echo "$changed_files" | grep -q "^\.eslintrc\.json$"; then
             yarn lint
           else
             echo "$changed_files" | grep -E "(.js$|.jsx$|.ts$|.tsx$)" | xargs -r yarn lint
           fi
 
       - name: Prettier
         run: |
-          changed_files=$(git diff --name-only --diff-filter=ACMRUXB origin/main)
+          changed_files=$(git diff --name-only --diff-filter=ACMRUXB origin/${{ github.base_ref }})
           
```

yamllint

[error] trailing spaces

(trailing-spaces)